ci/verify.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+./vendor/bin/php-cs-fixer fix ./src
+./vendor/bin/psalm
+./vendor/bin/phpcs --config-set colors 1
+./vendor/bin/phpcs --standard=PSR2 --error-severity=1 --standard=phpcs.xml ./src
+./vendor/bin/phpmd ./src text phpmd.xml
+phpdbg -qrr vendor/bin/phpunit --coverage-clover clover.xml
```

Comment:
you could also enable https://scrutinizer-ci.com for this project so that the coverage would be visible, along with many other useful details about the quality
example: https://scrutinizer-ci.com/g/xsolve-pl/salesforce-client/